index.html

```diff
@@ -27,21 +27,163 @@ <h4 class="dev">I’m a Software Developer</h4>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="button" class="project-btn">See Project</button>
+      </div>
+      <div class="project">
+        <h3>Profesional Art Printing Data</h3>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="button" class="project-btn">See Project</button>
+      </div>
+      <div class="project">
+        <h3>Profesional Art Printing Data</h3>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="button" class="project-btn">See Project</button>
+      </div>
+      <div class="project">
+        <h3>Profesional Art Printing Data</h3>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="button" class="project-btn">See Project</button>
+      </div>
+    </section>
+    <section class="abouts">
+      <div class="about">
+        <h3>About me</h3>
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <button type="button">Get My Resume</button>
+        <hr />
+        <ul>
+          <li class="lang">
+            <a href="#"><img src="./img/Rectangle55.png" alt="" /></a>Language
+          </li>
+          <li>Javascript</li>
+          <li>Ruby</li>
+          <li>Html</li>
+          <li>Css</li>
+        </ul>
+        <ul>
+          <li class="frame">
+            <a href="#"><img src="./img/Rectangle56.png" alt="" /></a>Frameworks
+          </li>
+          <li>Bootstrap</li>
+          <li>Ruby on Rails</li>
+          <li>RSPec</li>
+          <li>CapyBara</li>
+          <li>Selenium</li>
+        </ul>
+        <ul>
+          <li class="skill">
+            <a href="#"><img src="./img/Rectangle57.png" alt="" /></a> Skills
+          </li>
+          <li>Codekit</li>
+          <li>Github</li>
+          <li>Codepen</li>
+          <li>Gitlab</li>
+          <li>Terminal</li>
+        </ul>
```

Comment:
- [ ] Good job implementing this, please make the technologies in each category a list `ul` as well and each technology would be a list item `li`. 😁